src/configuration.cc

```diff
@@ -231,9 +233,12 @@ void Configuration::ParseConfiguration(std::istream& input) {
   metadata_reporter_user_agent_ =
       config["MetadataReporterUserAgent"].as<std::string>(
           metadata_reporter_user_agent_);
-  metadata_ingestion_endpoint_format_ =
-      config["MetadataIngestionEndpointFormat"].as<std::string>(
-          metadata_ingestion_endpoint_format_);
+  metadata_ingestion_host_ =
+      config["MetadataIngestionHost"].as<std::string>(
```

Comment:
This makes the change not backwards-compatible, and thus this is no longer a drop-in replacement for the v1beta2 version.